charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.8.0
+version: 12.8.1
```

Comment:
seems you might need to bump the chart version as well.

Reply:
bump version to resolve conflit